Merging relnotes for October release (#581)
MicrosoftDocs/quantum-docs-pr: articles/relnotes/index.md

For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).

## Version 0.x

*Release date: October 30th, 2019*

This release contains the following:

> - IQ# and Python host functionality can now be installed using conda

Reviewer: Should be removed.

> - Updates to the Python installation, Python can now be installed using conda

Reviewer 2: ```suggestion
- IQ# and Python host functionality can now be installed using conda
```

> - Updates to Visual Studio Code & Visual Studio extensions to include language server support 

Reviewer 2: Language server support has been there for a while, but with 0.10, the language server is now deployed as a self-contained executable such that the VS Code extension does not require the .NET Core SDK to be installed.

> - Migration to .NET 3.0

Reviewer 2: ```suggestion
- Migration to .NET Core 3.0
```

> - Updates to Visual Studio Code & Visual Studio extensions to deploy language server as a self-contained executable, no longer requiring the .NET Core SDK to be installed  

Reviewer 3: The "no longer requiring the .NET Core Sdk to be installed" should be replace by "eliminating the .NET Core Sdk version dependency" (meaning any version will work - which had been the problem before when people needed to update their dotnet version every time we updated ours), but they still need something installed.

- Breaking change to Microsoft.Quantum.Simulation.Core.IOperationFactory with introduction of new `Fail` method. It affects only custom simulators that do not extend SimulatorBase. For more details, [view the pull request on GitHub](https://github.com/microsoft/qsharp-runtime/pull/59).

> 

Reviewer 3: Do you want to add that we actually support Deprecated attributes? Not that there are a lot of people maintaining Q# packages where it would make sense to properly deprecate stuff - but maybe we should not assume that in our release notes.

Author: sounds good, I'll add it.
